src/assets/Azure.Core.Shared/OperationHelpers.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+#pragma warning disable CS8600, CS8604, CS8605
+
+namespace Azure.Core
+{
+    internal static class OperationHelpers
+    {
+        public static TimeSpan DefaultPollingInterval { get; } = TimeSpan.FromSeconds(1);
+
+        public static T GetValue<T>(ref T? value) where T: class
```

Comment:
Not sure I like to have an extension method on every value. Can we get rif of it?

Reply:
Oh, it's not an extension method, my bad.

Reply:
This isn't my code. This is Azure.Core. It is a downloaded file.